src/webui/components/common/OrderAndAddRow.tsx

```diff
@@ -0,0 +1,26 @@
+
+import johnDoe from '../../../domain/authentication/johnDoe';
+import create from '../../../domain/reaction/create';
+import { ClickArea, Icon, Row } from '../../designsystem/module';
+import OrderSelection from './elementary/OrderSelection';
+
+export type Props = {
+    selected?: 'recent' | 'popular';
+    postId: string;
+    orderChangeHandler?: (oldKey: string, newKey: string) => void;
+};
+
+export default function Component({ selected, postId, orderChangeHandler }: Props)
+{
+    const clickHandler = () => 
+    {
+        create(johnDoe, postId, `Dit is een random comment ${Math.random() * 1000}`);
```

Comment:
Why does this common component execute specific domain logic? 😲 Domain logic should only be called from features.

Also: what does 'Dit is een random comment' mean? Looks like a foreign language to me... 😉